src/css/theme/_search-box.scss

```diff
@@ -1,15 +1,13 @@
 .ais-search-box {
   display: inline-block;
   position: relative;
-  max-width: 300px;
-  width: 100%;
```

Comment:
How to make sure we don't break current implementations with this change?

Reply:
this is a duplicate declaration, already set in the main css